.changeset/tall-moments-behave.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/factory': minor
+---
+
+Improved the Factory audit log with a density timeline, category filters, responsive rows, and automatic history loading. Intake binding changes now appear in the affected project's audit history.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)